client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -101,6 +101,15 @@
     @Option(names = "--password", paramLabel = "<password>", description = "Prompt for password")
     public boolean password;
 
```

Comment:
commit message: `Add ExternalAuthorizert to cli` -> `Add ExternalAuthorizer to cli`
Why `Authorizer`? in code used `Authentication`

Reply:
Maybe let's focus on user visible change in commit title, not impl detail. Maybe `Support external authentication in CLI`?

Reply:
good idea.